<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserMain</class>
 <widget class="QWidget" name="UserMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_current_dir">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <underline>false</underline>
           </font>
          </property>
          <property name="text">
           <string>文件列表</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="list_files">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget2" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>文件属性</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="text_attributes">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:13pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_5" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btn_create_file">
            <property name="text">
             <string>新建文件</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_open_file">
            <property name="text">
             <string>打开文件</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_read_file">
            <property name="text">
             <string>读出文件</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_write_file">
            <property name="text">
             <string>写入文件</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_rename">
            <property name="text">
             <string>重命名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_delete">
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btn_create_dir">
            <property name="text">
             <string>新建目录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_open_dir">
            <property name="text">
             <string>进入目录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_return_pre">
            <property name="text">
             <string>返回上级目录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_return_home">
            <property name="text">
             <string>返回主目录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_search">
            <property name="text">
             <string>在当前目录查找</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
